packages/mask/src/plugins/Collectible/SNSAdaptor/Collectible.tsx

```diff
@@ -207,63 +207,80 @@ export function Collectible(props: CollectibleProps) {
         return tabMap[tabIndex] || null
     }
 
-    const endDate = asset.value?.end_time
+    const _asset = asset.value
+    const endDate = _asset.end_time
     return (
         <>
             <CollectibleCard classes={classes}>
                 <CardHeader
                     avatar={
-                        <Link
-                            href={asset.value.owner?.link}
-                            title={asset.value.owner?.user?.username ?? asset.value.owner?.address ?? ''}
-                            target="_blank"
-                            rel="noopener noreferrer">
-                            <Avatar src={asset.value.owner?.profile_img_url ?? ''} />
-                        </Link>
+                        (_asset.collectionLinkUrl && (
```

Comment:
Make a component for this just like: packages/mask/src/plugins/Trader/SNSAdaptor/trending/Linking.tsx.